setup/config.ts

```diff
@@ -1,15 +1,30 @@
-export const networks = {
+declare type Network = {
+    /**
+     * Base URL of the network
+     */
+    base: string,
+
+    /**
+     * Path to a resource, either absolute or relative
+     */
+    icon?: any,
```

Comment:
This can be type string right?

Reply:
Good catch. It was but due to issues with the icons I changed it to test something. I'll revert it back to `string`